services/core-api/app/api/utils/helpers.py

```diff
@@ -1,6 +1,21 @@
 import re
 
+
 def clean_HTML_string(raw_html):
-  cleanr = re.compile('<.*?>')
-  cleantext = re.sub(cleanr, '', raw_html)
-  return cleantext
\ No newline at end of file
+    cleanr = re.compile('<.*?>')
+    cleantext = re.sub(cleanr, '', raw_html)
+    return cleantext
+
+
+"""
```

Comment:
Can you please move this comment into or closer to the function?

Reply:
Sure Luke :)